drivers/dma/CMakeLists.txt

```diff
@@ -10,3 +10,4 @@ zephyr_library_sources_ifdef(CONFIG_DMA_DW		dma_dw.c)
 zephyr_library_sources_ifdef(CONFIG_DMA_NIOS2_MSGDMA	dma_nios2_msgdma.c)
 zephyr_library_sources_ifdef(CONFIG_DMA_SAM0		dma_sam0.c)
 zephyr_library_sources_ifdef(CONFIG_USERSPACE		dma_handlers.c)
+zephyr_library_sources_ifdef(CONFIG_DMA_IMX_RT		dma_imx_rt.c)
```

Comment:
Can we call this `dma_mcux_edma.c` instead? Is the EDMA on imx rt similar enough to the EDMA on kinetis that we could use the same driver?

Reply:
let me check the clock settings on kinetis, and have a test on frdmk64f

Reply:
tested on frdmk64f, the driver can be shared with dts config, but the test application need some changes, I will provide a solution for review soon